ape_etherscan/client.py

```diff
@@ -45,6 +45,10 @@ def get_etherscan_uri(ecosystem_name: str, network_name: str):
             if network_name == "mainnet"
             else "https://mumbai.polygonscan.com"
         )
+    elif ecosystem_name == "avalanche":
+        return (
```

Comment:
Unnecessary parentheses and indentation here; our linters may be complaining about this.
You can do `return  "https://snowtrace.io"` on one line to fix.

Reply:
updated [22c3385](https://github.com/ApeWorX/ape-etherscan/pull/46/commits/22c3385a96243240125044bd7e70ce36f6fd5afa)